Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -5248,7 +5251,9 @@ void WebPageProxy::scalePageInViewCoordinates(double scale, const IntPoint& cent
     if (!hasRunningProcess())
         return;
 
-    send(Messages::WebPage::DidScalePageInViewCoordinates(scale, centerInViewCoordinates));
+    forEachWebContentProcess([&] (auto& process, auto pageID) {
+        process.send(Messages::WebPage::DidScalePageInViewCoordinates(scale, centerInViewCoordinates), pageID);
```

Comment:
This change doesn’t look like it does anything? The receiver still has a local mainframe downcast and early return.

Reply:
This is... a good point. Fixed that :)